SEED-platform/seed
# 2179 api v3 dq

#### Any background context you want to provide?
The previous Data Quality Check endpoints worked but could be improved upon in terms of "RESTful-ness" and DRF serializer usage.

#### What's this PR do?
Build out V3 Data Quality Check and Data Quality Check Rule endpoints per the attached issue. 

Start using nested_routers to separate Rule resource. Start using Rule serializer to allow the usage of auto-generated Django Rest Framework endpoints.

#### How should this be manually tested?
"Try" the API V3 Data Quality Check and Data Quality Check Rule endpoints. 

Code review can be done by viewing commits in order.

#### What are the relevant tickets?
#2179 

#### Screenshots (if appropriate)
